uavcan/transport.py

```diff
@@ -490,6 +489,27 @@ def source_node_id(self, value):
         self._set_field("source_node_id", value)
 
     @property
+    def isMessage(self):
+        return self.transfer_priority != TransferPriority.SERVICE
+
+    @property
+    def isService(self):
+        return self.transfer_priority == TransferPriority.SERVICE
+
+    @property
+    def isServiceRequest(self):
+        return self.isService and self.request_not_response
+
+    @property
+    def isServiceResponse(self):
+        return self.isService and not self.request_not_response
+
+    @property
+    def isBroadcast(self):
+        return self.isMessage and self.broadcast_not_unicast
+
```

Comment:
Improper coding style in the above methods.

1. According to [PEP8](https://www.python.org/dev/peps/pep-0008/), attributes should be named `in_lower_case_with_underscores`.
2. Normally, property names should not be verbs, so `is` prefix should be removed.

It seems redundant to implement properties for both service response and service request, since they negate each other.